SorterOfPersonsByAge/src/Persons.java

```diff
@@ -0,0 +1,19 @@
+public class Persons {
+    String firstName;
```

Comment:
Always use access modifiers with reduced visibility.
Otherwise use javadoc to explain why the data is visible.